### PR TITLE
Complete three column layout challenge v1.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
Screenshot 2023-07-22 at 4 46 12 PM

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  </div>
+  <div class="column-one-third">
+      <h3>Integrates with existing tools</h3>
+      <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+  </div>
+    <div class="column-one-third">
+      <h3>In-depth Onboarding</h3>
+      <p>We train your entire team and provide you with support when and as it's needed</p>
+    </div>
+</div--->
```
Comment (shawnkost) [inline css-three-column-layout/index.html]: Delete all old code

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+header {
+  width: 100%;
+  font-family: Lato, sans-serif;
+  font-size: larger;
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 25px;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: If you're using this to add space between the `h2` and the content below, you should be using `margin-bottom` instead. `margin` is used to add space between elements

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+div {
+  max-width: 100%;
+}
+
+img {
+  width: 25%;
+  border-radius: 50%;
+}
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: You're immediately overriding these styles right below this on lines 45 - 51. If none of these styles are being applied, why include them?

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few tweaks here Dennis! Also one other thing I noticed

1. There should be less space in-between the text underneath your image. Your "Streamlined QMS" & the paragraph text underneath it for example. Compare your page to our example image. Ours has the text closer together

**Comment by dennis-pnguyen** [comment]
Screenshot 2023-07-24 at 9 26 10 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**